testkit/src/main/java/org/apache/calcite/test/SqlOperatorTest.java

```diff
@@ -1418,6 +1418,14 @@ protected static Calendar getCalendarNotTooNear(int timeUnit) {
     }
   }
 
+  /** Test case for
+   * <a href="https://issues.apache.org/jira/browse/CALCITE-6168">[CALCITE-6168]
```

Comment:
I don't think this test belongs here, it's not a test case for a specific operator. I think there is a test class called `RexExecutorTest` that might be better.